milton-processor/index.ts

```diff
@@ -1,40 +1,54 @@
-// Copyright 2018 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the 'License');
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an 'AS IS' BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 import express = require('express');
 import jsdomLib = require('jsdom');
 import domPurify = require('dompurify');
-import Readability = require("readability");
+import Readability = require('readability');
+import ajax = require("xmlhttprequest");
 import fs = require('fs');
 
 const PORT = Number(process.env.PORT) || 8080;
 const app = express();
 const JSDOM = jsdomLib.JSDOM;
 
-const secret = fs.readFileSync('secret', 'utf8');
+const serverSecret = fs.readFileSync('secret', 'utf8').trim();
+function authenticated(secret: string): boolean {
+  return secret == serverSecret
```

Comment:
I realize it makes no practical difference here, but I still think I prefer the always-use-`===` style